### ISSUE TITLE
Highlights broken after an import without semi-colon

### ISSUE THREAD

**Issue by abe33** [atom/language-javascript#478]
### Description

If an import statement at the top of the file isn't ended with a semi-colon the whole file highlighting gets broken.

Without semi-colons:
![no-semi-colon](https://cloud.githubusercontent.com/assets/247140/21587768/33d83188-d0e0-11e6-861a-c4a616c0503e.png)

With semi-colons:
![semi-colon](https://cloud.githubusercontent.com/assets/247140/21587769/36a1eaa8-d0e0-11e6-99e3-1c0a12afb38e.png)

### Versions

```
> atom --version
Atom    : 1.14.0-dev-5669246
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0

> apm --version
apm  1.15.1
npm  3.10.5
node 4.4.5 x64
python 2.7.10
git 2.2.1
```

**Comment by MaximSokolov** [comment]
This is the expected behavior. According to the specification, a terminating semicolon is needed, so [it was decided](https://github.com/atom/language-javascript/issues/460) to require it in favor of supporting multiline import statements. I guess an issue should be raised in the [Babel repository](https://github.com/babel/babel/issues).

**Comment by maxbrunsfeld** [comment]
Reopening because we need to fix this. See https://github.com/atom/language-javascript/issues/460#issuecomment-270017461.

**Comment by 50Wliu** [comment]
Closed by #480.
